README.md

```diff
@@ -15,9 +15,14 @@ npm install react-chrome-redux
 
 ## Overview
 
-`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a single place to store a single source of truth redux Store.
+`react-chrome-redux` aims to do one simple thing for Chrome extensions: treat the background script as a place to store a single source of truth redux Store.
```

Comment:
I feel like we can weave in a bit of the philosophy discussed in the introduction and #34. What are your thoughts?

Something like:

> `react-chrome-redux` allows you to build your Chrome extension like a Redux-powered webapp. The background page holds the Redux store, while Popovers and Content-Scripts act as UI Components, passing actions and state updates between themselves and the background store. At the end of the day, you have a single source of truth (your Redux store) that describes the entire state of your extension.